src/OpenTelemetry.Exporter.Jaeger/Implementation/JaegerActivityExtensions.cs

```diff
@@ -26,7 +26,8 @@ internal static class JaegerActivityExtensions
         private static readonly Dictionary<string, int> PeerServiceKeyResolutionDictionary = new Dictionary<string, int>(StringComparer.OrdinalIgnoreCase)
         {
             [SpanAttributeConstants.PeerServiceKey] = 0, // peer.service primary.
-            ["net.peer.name"] = 1, // peer.service first alternative.
+            [SpanAttributeConstants.NetPeerName] = 1, // peer.service first alternative.
```

Comment:
one first and three second alternatives =)

Reply:
Hah ok yes, the comments suck. It's more like tiers. I was trying to have the official named attributes called out in the spec be taken first, in case multiple matches are found on a span. Spec PR (https://github.com/open-telemetry/opentelemetry-specification/pull/534) never gained much traction.

Reply:
Updated the comments.